src/modules/Builder/features/Activities/Activities.tsx

```diff
@@ -12,17 +12,18 @@ import { getNewActivity } from 'modules/Builder/pages/BuilderApplet/BuilderApple
 import { BuilderContainer } from 'shared/features';
 
 import { Item } from '../../components';
-import { getActions } from './Activities.const';
 import { ActivitiesHeader } from './ActivitiesHeader';
+import { getActions } from './Activities.const';
 
 export const Activities = () => {
   const { t } = useTranslation('app');
-  const { control, watch } = useFormContext();
+  const { control, watch, getFieldState } = useFormContext();
   const navigate = useNavigate();
   const { appletId } = useParams();
-  const [deletionModalVisible, setDeletionModalVisible] = useState(false);
+  const [activityToDelete, setActivityToDelete] = useState<string | undefined>('');
```

Comment:
why did you use `undefined` ?
```suggestion
  const [activityToDelete, setActivityToDelete] = useState<string>('');
```

Reply:
fixed